arch/arm/src/s32k1xx/s32k1xx_enet.c

```diff
@@ -1217,8 +1217,8 @@ static int s32k1xx_ifup_action(struct net_driver_s *dev, bool resetphy)
   int ret;
 
   ninfo("Bringing up: %d.%d.%d.%d\n",
-        dev->d_ipaddr & 0xff, (dev->d_ipaddr >> 8) & 0xff,
-        (dev->d_ipaddr >> 16) & 0xff, dev->d_ipaddr >> 24);
+        (int)(dev->d_ipaddr & 0xff), (int)((dev->d_ipaddr >> 8) & 0xff),
+        (int)((dev->d_ipaddr >> 16) & 0xff), (int)(dev->d_ipaddr >> 24));
```

Comment:
Let's go with `"Bringing up: %u.%u.%u.%u\n"` instead of adding `(int)`

Reply:
Even we change to %u, the cast is still needed since [u]int32_t map to [unsigned ]long int not [unsigned ]int.

Reply:
Agree